src/common/store/commonReduxStoreReducer.js

```diff
@@ -0,0 +1,21 @@
+export const types = {
+  OBSERVATIONS_VALUE: `OBSERVATIONS_VALUE`
```

Comment:
Would not ADD_CONCEPT be a better name for this type instead of OBSERVATIONS_VALUE, as it is actually being used to add a concept to the store and not the observation.